modules/ROOT/pages/audit-logging.adoc

```diff
@@ -1138,6 +1138,41 @@ Environments that are created automatically when you create an organization or b
 
 |===
 
+== Anypoint DataGraph
```

Comment:
Should we just say DataGraph instead of Anypoint DataGraph as in the audit logs table it just says DataGraph.